graphql/resolvers.go

```diff
@@ -942,6 +942,17 @@ func (r *queryResolver) Patch(ctx context.Context, id string) (*restModel.APIPat
 	if err != nil {
 		return nil, InternalServerError.Send(ctx, err.Error())
 	}
+	tasks, _, err := r.sc.FindTasksByVersion(id, []string{}, []string{}, "", "", 0, 0, []string{task.DisplayStatusKey}, []task.TasksSortOrder{})
```

Comment:
Should check this error

Reply:
Is it worth just filtering for TaskAborted and TaskFailureStatuses here?

Reply:
Nice catch just updated!